Networking/Networking/Remote/CouponsRemote.swift

```diff
@@ -188,8 +188,14 @@ public final class CouponsRemote: Remote, CouponsRemoteProtocol {
     public func loadCouponReport(for siteID: Int64,
                                  couponID: Int64,
                                  completion: @escaping (Result<CouponReport, Error>) -> Void) {
-        let parameters = [
-            ParameterKey.coupons: [couponID]
+        // Get an "ancient" date to fetch all possible reports
+        let startTime = Date(timeIntervalSince1970: 1)
+        let dateFormatter = ISO8601DateFormatter()
+        let formattedTime = dateFormatter.string(from: startTime)
+
+        let parameters: [String: Any] = [
+            ParameterKey.coupons: [couponID],
+            ParameterKey.after: formattedTime
```

Comment:
Is it possible to remove this parameter or is it mandatory?
BTW, if it's mandatory, I would say to insert the parameter among those of the method, setting the start time configured as above as the default value. Maybe in the future, we will need to filter the report by date.

Reply:
Thank you - this is the better way to do it because the lower layer should not need to care about the app's business logic. I updated this in 586e6fa71639d782923c8f9cfb1e2f83a709754f.